scipy/optimize/tests/test_zeros.py

```diff
@@ -19,6 +19,20 @@
 
 
 class TestBasic(object):
+    def run_check_by_name(self, name, smoothness=0):
+        a = .5
+        b = sqrt(3)
+        xtol = 4*np.finfo(float).eps
+        rtol = 4*np.finfo(float).eps
```

Comment:
perhaps use [`TOL` from line 18](../../blob/8b1addf06136a47280eaabb52a7e5e8113f6c6b7/scipy/optimize/tests/test_zeros.py#L18)?

Reply:
Why not use `TOL` same value, sound it's already defined on L21? Sorry to comment so late, feel free to ignore. :)

Reply:
thanks @mikofski, incorrectly resolved this one. it's a tiny thing so I'm going to spare the CI another re-run, apologies